pkg/pubsub/client/pubsub_client.go

```diff
@@ -0,0 +1,80 @@
+package client
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/open-policy-agent/gatekeeper/pkg/pubsub/connection"
+	"github.com/open-policy-agent/gatekeeper/pkg/pubsub/dapr"
+)
+
+func init() {
+	flag.Var(pubSubs, "pub-sub-tool", "Preferred pub-sub tool, e.g dapr, rabbitmq. This flag can be declared more than once. Omitting will default to not using any pub sub tool.")
```

Comment:
instead of `tool` should we say "service", or "provider"?

Reply:
+1